### PR TITLE
Feature/Rework main tabs according new design using svg icons (#1748)

### LINKED ISSUE #1748: Rework main tabs according new design using svg icons
image

### PR DESCRIPTION
**Pull request by flexsurfer**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)
fixes #1748

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready

### CODE REVIEW

**Review by janherich — APPROVED** [review]

File: src/status_im/components/styles.cljs [hunk 1]
```diff
@@ -19,6 +19,7 @@
 (def color-gray5 "#d9dae1")
 (def color-gray6 "#212121")
 (def color-gray7 "#9fa3b4")
+(def color-gray8 "#6E777E")
```
Comment (jeluard) [inline src/status_im/components/styles.cljs:22]: Wow 8 different grays :)

File: src/status_im/components/tabs/styles.cljs [hunk 2]
```diff
@@ -1,29 +1,24 @@
 (ns status-im.components.tabs.styles
-  (:require [status-im.components.styles :as st]
+  (:require-macros [status-im.utils.styles :refer [defnstyle defstyle]])
+  (:require [status-im.components.styles :as common-styles]
```
Comment (jeluard) [inline src/status_im/components/tabs/styles.cljs]: What about using `styles` ? I started to migrate to it.

File: src/status_im/components/tabs/styles.cljs [hunk 3]
```diff
@@ -1,29 +1,24 @@
...
    :right            0
    :height           tabs-height
-   :background-color st/color-white
+   :background-color common-styles/color-white
    :transform        [{:translateY 1}]})
 
 (def tabs-container-line
   {:border-top-width 1
-   :border-top-color "#D7D7D7"})
+   :border-top-color "#e8ebec"})
```
Comment (jeluard) [inline src/status_im/components/tabs/styles.cljs]: Can this be a constant?

File: src/status_im/components/styles.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/components/styles.cljs:22]: #TooManyGraysException

File: src/status_im/components/tabs/views.cljs [hunk 4]
```diff
@@ -1,32 +1,21 @@
...
-  (let [active?   (= view-id selected-view-id)]
-    [touchable-highlight {:style    st/tab
-                          :disabled active?
-                          :on-press  #(dispatch [:navigate-to-tab view-id])}
-     [view {:style st/tab-container}
+(defn- tab [{:keys [view-id title icon-active icon-inactive selected-view-id]}]
+  (let [active? (= view-id selected-view-id)]
+    [react/touchable-highlight {:style    styles/tab
+                                :disabled active?
+                                :onPress  #(dispatch [:navigate-to-tab view-id])}
```
Comment (jeluard) [inline src/status_im/components/tabs/views.cljs]: No more camel-case?

File: src/status_im/components/styles.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/components/styles.cljs:22]: tbh i don't know what to do with that. and i don't like it, we need some styles sheet from designers with color names

File: src/status_im/ui/screens/views.cljs [hunk 5]
```diff
@@ -60,12 +60,10 @@
     (when view-id
       (let [current-view (validate-current-view view-id signed-up?)]
         (let [component (case current-view
-                          :wallet main-tabs
+                          (:wallet :chat-list :discover :contact-list) main-tabs
```
Comment (jeluard) [inline src/status_im/ui/screens/views.cljs:63]: Wow never used this syntax.

**Review by jeluard — CHANGES_REQUESTED** [review]

File: src/status_im/components/tabs/styles.cljs (hunk 2 above)
Comment (flexsurfer) [reply src/status_im/components/tabs/styles.cljs]: do you mean `styles` alias instead `common-styles` ?

File: src/status_im/components/tabs/styles.cljs (hunk 3 above)
Comment (flexsurfer) [reply src/status_im/components/tabs/styles.cljs]: maybe :)

File: src/status_im/components/tabs/views.cljs (hunk 4 above)
Comment (flexsurfer) [reply src/status_im/components/tabs/views.cljs]: do you mean, i should use kebab here? ok

File: src/status_im/components/styles.cljs (hunk 1 above)
Comment (jeluard) [reply src/status_im/components/styles.cljs:22]: :+1:

File: src/status_im/components/tabs/styles.cljs (hunk 2 above)
Comment (jeluard) [reply src/status_im/components/tabs/styles.cljs]: Yes

File: src/status_im/components/tabs/views.cljs (hunk 4 above)
Comment (jeluard) [reply src/status_im/components/tabs/views.cljs]: Yes

File: src/status_im/components/tabs/styles.cljs (hunk 3 above)
Comment (flexsurfer) [reply src/status_im/components/tabs/styles.cljs]: gray9 ?

**Comment by flexsurfer** [comment]
@jeluard done

**Review by jeluard — APPROVED** [review]

**Review by rasom — APPROVED** [review]

**Pull request merged**